src/numbers.js

```diff
@@ -1,45 +1,60 @@
 function add (a, b) {
   // your code here
+  return a + b;
 }
 
 function subtract (a, b) {
   // your code here
+  return a - b;
 }
 
 function multiply (a, b) {
   // your code here
+  return a * b;
 }
 
 function divide (a, b) {
   // your code here
+  return a / b;
 }
 
 function power (a, b) {
   // your code here
+  return a ** b;
 }
 
 function round (a) {
   // your code here
+  return Math.round(a);
 }
 
 function roundUp (a) {
   // your code here
+  return Math.ceil(a);
 }
 
 function roundDown (a) {
   // your code here
+  return Math.floor(a);
 }
 
 function absolute (a) {
   // your code here
+  return Math.abs(a);
 }
 
 function quotient (a, b) {
   // your code here
+  let quotient2 = Math.floor(a / b);
+  if (a <0 && b >0 || a > 0 && b < 0) {
+     quotient2 += 1;
+  }
+  return quotient2;
```

Comment:
You gave this is a really good go! As it happens theres a built in function for this: Math.trunc (see [docs here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/trunc#:~:text=The%20Math.,by%20removing%20any%20fractional%20digits.))
```
  return Math.trunc(a / b);
```